### PR TITLE
chore(zh-cn): translate for InputEvent.inputType

### PR DESCRIPTION
**Pull request by fuchunhui**
### Description
1. fix link error.
2. translate content.

### CODE REVIEW

File: files/zh-cn/web/api/inputevent/data/index.md [hunk 1]
```diff
@@ -1,27 +1,21 @@
...
-
-### 返回值
-
-一个 {{domxref("DOMString")}}。
+一个字符串。
 
 ## 示例
 
-在下面的简单示例中，我们在 [input](/zh-CN/docs/Web/API/Element/input_event) 事件上设置了一个事件监听器，以便在对 {{htmlelement("input")}} 元素的内容进行任何更改时（通过键入或粘贴），通过 **`InputEvent.data`** 属性检索添加的文本，并在 `<input>` 下面的段落中报告。
+在下面的简单示例中，我们在 [input](/zh-CN/docs/Web/API/Element/input_event) 事件上设置了一个事件监听器，**`InputEvent.data`** 检索 {{htmlelement("input")}} 元素的任何文本变化，使用 [`Node.textContent`](/zh-CN/docs/Web/API/Node/textContent) 属性将其插入到段落中。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/api/inputevent/data/index.md:18]:
```suggestion
在下面的简单示例中，我们在 [input](/zh-CN/docs/Web/API/Element/input_event) 事件上设置了一个事件监听器，任何对 {{htmlelement("input")}} 元素的文本更改都会被 `InputEvent.data` 获取，使用 [`Node.textContent`](/zh-CN/docs/Web/API/Node/textContent) 属性将其插入到段落中。
```

File: files/zh-cn/web/api/inputevent/data/index.md [hunk 2]
```diff
@@ -32,11 +26,11 @@ var string = inputEvent.data;
...
 
 ```js
-var editable = document.querySelector("input");
-var result = document.querySelector(".result");
+const editable = document.querySelector("input");
+const result = document.querySelector(".result");
 
 editable.addEventListener("input", (e) => {
-  result.textContent = "Inputted text: " + e.data;
+  result.textContent = `Inputted text: ${e.data}`;
```
Comment (JasonLamv-t) [inline files/zh-cn/web/api/inputevent/data/index.md:33]:
```suggestion
  result.textContent = `输入的文字：${e.data}`;
```

File: files/zh-cn/web/api/inputevent/datatransfer/index.md [hunk 3]
```diff
@@ -1,27 +1,23 @@
...
-```
-
-### 返回值
+## 值
 
 一个 {{domxref("DataTransfer")}} 对象。
 
 ## 示例
 
 在下面的简单示例中，我们在 [input](/zh-CN/docs/Web/API/Element/input_event) 事件上设置了一个事件监听器，以便在将任何内容粘贴到 {{htmlelement("p")}} 元素时，通过 [`InputEvent.dataTransfer.getData()`](/zh-CN/docs/Web/API/DataTransfer/getData) 方法检索其 HTML 源代码，并在输入框下面的段落中报告。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/api/inputevent/datatransfer/index.md:18]:
```suggestion
在下面的简单示例中，我们在 [input](/zh-CN/docs/Web/API/Element/input_event) 事件上设置了一个事件监听器，以便在将任何内容粘贴到可编辑的 {{htmlelement("p")}} 元素时，通过 [`InputEvent.dataTransfer.getData()`](/zh-CN/docs/Web/API/DataTransfer/getData) 方法获取其 HTML 源代码，并将其报告在输入框下面的段落中。
```

File: files/zh-cn/web/api/inputevent/datatransfer/index.md [hunk 4]
```diff
@@ -1,27 +1,23 @@
...
 
-尝试复制并粘贴提供的部分内容以查看效果。注意，部分浏览器对其支持不佳。
+尝试复制并粘贴提供的部分内容以查看效果。
 
 ```html
 <p><span style="font-weight: bold; color: blue">Whoa, bold blue text!</span></p>
```
Comment (JasonLamv-t) [inline files/zh-cn/web/api/inputevent/datatransfer/index.md:23]: Please translate the content in HTML code if possible.

File: files/zh-cn/web/api/inputevent/inputevent/index.md [hunk 5]
```diff
@@ -1,38 +1,46 @@
...
+title: InputEvent：InputEvent() 构造函数
 slug: Web/API/InputEvent/InputEvent
+l10n:
+  sourceCommit: eab4066e72d5478de920e4020e5db71214dcffa6
 ---
 
-{{APIRef("DOM Events")}}{{SeeCompatTable}}
+{{APIRef("UI Events")}}
 
 构造函数 **`InputEvent()`** 返回一个新创建的 {{domxref("InputEvent")}} 对象。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/api/inputevent/inputevent/index.md:10]:
```suggestion
**`InputEvent()`** 构造函数创建一个新的 {{domxref("InputEvent")}} 对象。
```

File: files/zh-cn/web/api/inputevent/inputtype/index.md [hunk 6]
```diff
@@ -1,21 +1,17 @@
...
-```js
-var aString = inputEvent.inputType;
-```
-
-### 返回值
-
-一个包含所做输入的类型的字符串。有许多可能的值，例如 `insertText`、`deleteContentBackward`、`insertFromPaste` 和 `formatBold`。有关可用输入类型的完整列表，请参阅 [Input Events Level 1 规范的属性部分](https://rawgit.com/w3c/input-events/v1/index.html#interface-InputEvent-Attributes)。
+一个包含所做输入的类型的字符串。有许多可能的值，例如 `insertText`、`deleteContentBackward`、`insertFromPaste` 和 `formatBold`。有关可用输入类型的完整列表，请参阅 [Input Events Level 2 规范的属性部分](https://w3c.github.io/input-events/#interface-InputEvent-Attributes)。
 
 ## 示例
```
Comment (JasonLamv-t) [inline files/zh-cn/web/api/inputevent/inputtype/index.md:17]: Ditto. Please translate the content in HTML code if possible.

**Review by JasonLamv-t — COMMENTED** [review]
Sorry for the late. Some suggestions for your reference.
/cc @yin1999 Can you check the title of these changes. I can't confirm if their format is ok.

**Review by yin1999 — APPROVED** [review]
LGTM for the changes of `title`s.

File: files/zh-cn/web/api/inputevent/datatransfer/index.md (hunk 4 above)
Comment (fuchunhui) [reply files/zh-cn/web/api/inputevent/datatransfer/index.md]: ✅In the example, retaining English is also reasonable.

Comment (fuchunhui) [reply files/zh-cn/web/api/inputevent/datatransfer/index.md]: I got it. https://github.com/mdn/translated-content/blob/main/docs/README.md#localizing-code-blocks

**Review by JasonLamv-t — APPROVED** [review]
LGTM. Good job! @fuchunhui

**Pull request merged**